examples/solids/README.rst

```diff
@@ -0,0 +1,84 @@
+libCEED: Solid Mechanics Example
+--------------------------------
+
+This page provides a description of the solid mechanics example for the
+libCEED library, based on PETSc.
+
+This code solves the steady-state static momentum balance equations using unstructured high-order finite/spectral element spatial discretizations.
+In this mini-app, we consider three formulations used in solid mechanics applications: linear elasticity, Neo-Hookean hyperelasticity at small strain, and Neo-Hookean hyperelasticity at finite strain.
+
+Build by using
+
+``make``
+
+and run with
+
+``./elasticity -mesh [.exo file] -degree [degree] -nu [nu] -E [E] [boundary options] -problem [problem type] -forcing [forcing] -ceed [ceed]``
+
+Runtime options
+---------------
+
+.. inclusion-marker-do-not-remove
+
+There are five required command line options and a variety of additional command line options for this mini-app.
+The four required command line options are :code:`-mesh`, :code:`-degree`, :code:`-E`, and :code:`-nu`. Additionally, at least one boundary condition must be set, using :code:`-bc_zero` or :code:`-bc_clamp`.
```

Comment:
Are these "primary" or "principal" command-line options?

Reply:
What about mandatory? I am trying to communicate that the code won't run without these options set.

Reply:
Oh, sure. I thought they had defaults based on earlier runs. How about skip the prose and just say the following options are mandatory and have a [list-table](https://docutils.sourceforge.io/docs/ref/rst/directives.html#list-table)?

Reply:
updated